### PR TITLE
0.19 support

### PR DESCRIPTION
**Pull request by lukewestby**
This renames all references to `elm-make` to `elm make` and adds the `--optimize` flag. Since 0.18's `elm` binary will run `elm-make` when `elm make` is invoked this upgrade _should_ work for both 0.18 and 0.19.

### CODE REVIEW

File: src/CliArguments.hs [hunk 1]
```diff
@@ -39,10 +40,12 @@ parser = do
        help "Path to config file.")
   debug <-
     switch (long "debug" <> short 'd' <> help "Run jetpack in debug mode.")
-  warn <- switch (long "warn" <> short 'w' <> help "Output elm make warnings.")
+  warn <-
```
Comment (stoeffel) [inline src/CliArguments.hs]: should we drop warn?

Comment (stoeffel) [reply src/CliArguments.hs]: It’s also not set anymore in compiler

File: src/CliArguments.hs [hunk 2]
```diff
@@ -60,6 +63,10 @@ parser = do
         if watch
           then True
           else warn
+    , optimize =
+        if optimize
```
Comment (stoeffel) [inline src/CliArguments.hs]: would be nice to use a custom type for this. I should have done that for the other flags as well.

File: src/CliArguments.hs (hunk 1 above)
Comment (stoeffel) [reply src/CliArguments.hs]: https://github.com/NoRedInk/jetpack/search?q=warn&unscoped_q=warn

**Comment by lukewestby** [comment]
@stoeffel I removed warnings and made the compile mode a union type. I also modified the npm package locally to point to my build and ran it on the monolith and it looks like the progress bar is correct and it fails correctly.

**Comment by lukewestby** [comment]
@stoeffel ready for a publish whenever you are!

**Pull request merged**